modules/getting-started-web-console-granting-permissions.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-web-console.adoc
+
+:_content-type: PROCEDURE
+[id="getting-started-web-console-granting-permissions_{context}"]
+= Granting view permissions
+
+{product-title} automatically creates a few special service accounts in every project. The default service account takes responsibility for running the Pods. {product-title} uses and injects this service account into every Pod that launches.
+
+The following procedure creates a RoleBinding for the default ServiceAccount. The service account communicates with the {product-title} API to learn about Pods, Services, and resources within the project.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console.
+* You have a deployed image.
+* You are in the *Administrator* perspective.
+* You must have `cluster-admin` privileges.
```

Comment:
```
You must have `cluster-admin` privileges
```
granting user view permission doesn't require `cluster-admin` , project admin user also able to do that

Reply:
but following steps are only applicable for `cluster-admin` users

Reply:
Resolved this by stating the following:

> * You must have `cluster-admin` or `project-admin` privileges.